modules/Rock.cmake

```diff
@@ -500,6 +504,18 @@ function(rock_prepare_pkgconfig TARGET_NAME DO_INSTALL)
     set(PKGCONFIG_CFLAGS ${${TARGET_NAME}_PKGCONFIG_CFLAGS})
     set(PKGCONFIG_LIBS ${${TARGET_NAME}_PKGCONFIG_LIBS})
 
+    if (DEFINED ROCK_PUBLIC_CXX_STANDARD)
+        set(cxx_standard ${ROCK_PUBLIC_CXX_STANDARD})
+    elseif (CMAKE_VERSION VERSION_LESS "3.1")
```

Comment:
The `ROCK_PUBLIC_CXX_STANDARD` variable sets the standard as exported in the .pc file, it does not change how the standard is handled internally to the package. It is meant as a way to use C++11 internally but have C++98 headers (and avoid propagating the C++11 choice downstream).

Note that for CMake >= 3.1, when ROCK_PUBLIC_CXX_STANDARD is not used, the code only relies on CMake's own way to handle the C++ standard (setting the CXX_STANDARD property on the targets).